### PR TITLE
New package: jaaa-0.9.2

### PR DESCRIPTION
**Pull request by bluntphenomena**
New libraries:
- clthreads
- clxclient

### CODE REVIEW

File: srcpkgs/clthreads/template [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+homepage="https://kokkinizita.linuxaudio.org/linuxaudio/"
+distfiles="https://kokkinizita.linuxaudio.org/linuxaudio/downloads/${pkgname}-${version}.tar.bz2"
+checksum=c659b14c0c4055c44432cb83060b95d30ae0c1ecc6f50d73968e239c100f7a31
+
+pre_build() {
+	sed -i '/march=native/d' Makefile
+}
+
+pre_install() {
+        # fix wrong path
```
Comment (maxice8) [inline srcpkgs/clthreads/template]: wont indent

File: srcpkgs/clxclient/template [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Template file for 'clxclient'
+pkgname=clxclient
+version=3.9.2
+revision=1
+build_wrksrc=source
+build_style=gnu-makefile
+make_install_args="LIBDIR=/usr/lib"
+hostmakedepends="pkgconf"
```
Comment (maxice8) [inline srcpkgs/clxclient/template]: should be pkg-config

File: srcpkgs/jaaa/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'jaaa'
+pkgname=jaaa
+version=0.9.2
+revision=1
+build_wrksrc=source
+build_style=gnu-makefile
+hostmakedepends="pkgconf"
+makedepends="zita-alsa-pcmi-devel clthreads-devel clxclient-devel
+ alsa-lib-devel fftw-devel jack-devel libX11-devel libXft-devel"
+depends="libfftw"
```
Comment (maxice8) [inline srcpkgs/jaaa/template]: libfftw may not be necessary here

File: srcpkgs/jaaa/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'jaaa'
+pkgname=jaaa
+version=0.9.2
+revision=1
+build_wrksrc=source
+build_style=gnu-makefile
+hostmakedepends="pkgconf"
```
Comment (maxice8) [inline srcpkgs/jaaa/template]: pkg-config

File: srcpkgs/clxclient/template [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+pre_build() {
+	sed -i '/march=native/d' Makefile
+	# fix wrong header import
+	vsed -i 's/<clxclient.h>/"clxclient.h"/g' enumip.cc
+}
+
+post_install() {
+	# add missing symlink
+	ln -s libclxclient.so.${version} ${PKGDESTDIR}/usr/lib/libclxclient.so.${version%%.*}
```
Comment (maxice8) [inline srcpkgs/clxclient/template]: can be DESTDIR instead of PKGDESTDIR

**Comment by maxice8** [comment]
Keep all changes in one commit for each package.

**Comment by bluntphenomena** [comment]
> Keep all changes in one commit for each package.

**Comment by bluntphenomena** [comment]
> Keep all changes in one commit for each package.

thank you so much for your help! this is my first push

**Comment by maxice8** [comment]
On Wed, Apr 03, 2019 at 03:41:27PM -0700, bluntphenomena wrote:
> Closed #10535.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/10535#event-2251313279

please stop closing and opening PRs and just use git to amend commits and squash them.

**Comment by bluntphenomena** [comment]
> On Wed, Apr 03, 2019 at 03:41:27PM -0700, bluntphenomena wrote: Closed #10535. -- You are receiving this because you commented. Reply to this email directly or view it on GitHub: [#10535 (comment)](https://github.com/void-linux/void-packages/pull/10535#event-2251313279)
> please stop closing and opening PRs and just use git to amend commits and squash them.

sorry that was not my intention

**Comment by bluntphenomena** [comment]
@maxice8 all checks have passed, can you approve it?

**Comment by maxice8** [comment]
no, the commits need to be reduced to only 3, 1 per package

**Pull request closed without merging**